content/id/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -0,0 +1,374 @@
+secara terpisah untuk suatu sumber daya yang ada, maka perlu membuat FlowSchemas terpisah yang sesuai dengan
+masing-masing kata kerja dari permintaan yang bermutasi dan yang tidak bermutasi tersebut.
+{{< /caution >}}
+
+Ketika jumlah permintaan masuk yang diserahkan kepada sebuah
+PriorityLevelConfiguration melebihi dari tingkat konkurensi yang diizinkan, 
+bagian `type` dari spesifikasinya menentukan apa yang akan terjadi pada permintaan selanjutnya.
+Tipe `Tolak` berarti bahwa kelebihan lalu lintas akan segera ditolak
+dengan kode kesalahan HTTP 429 (yang artinya terlalu banyak permintaan). Tipe `Antrian` berarti permintaan
+di atas batas tersebut akan mengantri, dengan teknik _sharding shuffle_ dan _fair queuing_ yang digunakan
+untuk menyelaraskan kemajuan antara _flow_ permintaan.
+
+Konfigurasi antrian memungkinkan mengatur algoritma _fair queuing_ untuk sebuah
+tingkat prioritas. Detail algoritma dapat dibaca di [proposal pembaharuan](#what-s-next), namun secara singkat:
+
+* Meningkatkan `antrian` berarti mengurangi tingkat tabrakan antara _flow_ yang berbeda, 
+  sehingga berakibat pada biaya untuk meningkatkan penggunaan memori. Nilai 1 di sini secara 
+  efektif menonaktifkan logika _fair-queuing_, tetapi masih mengijinkan permintaan untuk 
+  dimasukkan kedalam antrian.
+
+* Meningkatkan `queueLengthLimit` berarti memperbolehkan lonjakan yang lebih besar dari lalu lintas 
+  untuk berkelanjutan tanpa menggagalkan permintaan apa pun, dengan konsekuensi akan meningkatkan
+  latensi dan penggunaan memori.
+
+* Mengubah `handSize` berarti memperbolehkan kamu untuk menyesuaikan probabilitas tabrakan antara
+   _flow_ yang berbeda dan keseluruhan konkurensi yang tersedia untuk satu _flow_ tunggal 
+   dalam situasi beban berlebih.
+  
+    {{< note >}}
+     `HandSize` yang lebih besar membuat dua _flow_ individual berpeluang kecil untuk bertabrakan
+     (dan dimana _flow_ yang satu bisa membuat _flow_ yang lain menderita), tetapi akan lebih memungkinkan
+     bahwa _flow_ dalam jumlah kecil akan dapat mendominasi apiserver. `HandSize` yang lebih besar juga
+     berpotensi meningkatkan jumlah latensi yang diakibatkan oleh satu _flow_ lalu lintas tunggal 
+     yang tinggi. Jumlah maksimum permintaan dalam antrian yang diijinkan dari sebuah _flow_ tunggal 
+     adalah `handSize * queueLengthLimit`.
+    {{< /note >}}
+
+Berikut ini adalah tabel yang menunjukkan koleksi konfigurasi _shuffle sharding_ 
+yang menarik, dimana setiap probabilitas _mouse_ (_flow_ dengan intensitas rendah) 
+yang diberikan akan dimampatkan oleh _elephant_ (_flow_ dengan intensitas tinggi) dalam sebuah koleksi ilustratif 
+dari jumlah _elephant_ yang berbeda. Silahkan lihat pada
+https://play.golang.org/p/Gi0PLgVHiUg, yang digunakan untuk menghitung nilai-nilai dalam tabel ini.
+
+{{< table caption="Contoh Konfigurasi Shuffle Sharding" >}}
+|HandSize|     Queues|	1 elephant|		4 elephants|		16 elephants|
```

Comment:
nit: you can localize the column titles, including hand size. Use terms from gambling or card games.
nit: use gajah as well or in place of _elephant_, and tikus in place of _mouse_.

Reply:
I prefer to keep the terms because it is still a better interpretation. Also, `mouse` and `elephant` are well-known words.